src/utils/helpers.js

```diff
@@ -4,7 +4,7 @@ export function buildRelationshipData(type, ids) {
   let data = [];
 
   if (ids === null || ids.length === 0) {
-    return null;
+    return '[]';
```

Comment:
@outrunthewolf I'm told the API expects an empty array in `PUT` body if removing all relationships from a resource.

https://moltin.api-docs.io/v2/product-relationships/update-a-products-relationships-to-brands-1